docs/census.rst

```diff
@@ -0,0 +1,78 @@
+Census
+=============
+
+********
+Overview
+********
+
+Connects to the Census API--it has been tested with the ACS and Economic Survey endpoints.
```

Comment:
Can you add a link to the Census API here? Also, this is super nitpicky, but do you mind editing this so there's a period instead of a double dash between the two sentences? Sorry to be such a grammar nerd 🤓

Reply:
Looks great! Going to merge now.